### ISSUE TITLE
Can't run a single test method from Gradle 6.5

### ISSUE THREAD

**Issue by yevhenii-nadtochii** [kotest/kotest#1531]
I'm trying to run a single test method from Gradle and I can't. Using example from [gradle docs](https://docs.gradle.org/current/userguide/java_testing.html)
`gradle clean test`
```
worder.StampedSourceFileTest > should work with valid stamp PASSED

worder.StampedSourceFileTest > should work with files with no stamp PASSED

worder.StampedSourceFileTest > should not work with invalid stamp PASSED

worder.StampedSourceFileTest > should retrieve file's creation date PASSED
```
But
`gradle test --tests "worder.StampedSourceFileTest.should retrieve file's creation date"`
```
Execution failed for task ':test'.
> No tests found for given includes: [worder.StampedSourceFileTest.should retrieve file's creation date](--tests filter)
```
Kotest: io.kotest:kotest-runner-junit5-jvm:4.1.1

**Comment by sksamuel** [comment]
What's the value of your --tests
Is it `worder.StampedSourceFileTest.should retrieve file's creation date`
And is worder.StampedSourceFileTest the FQN for your test class ?

**Comment by yevhenii-nadtochii** [comment]
Yes, it is. I've put output of `gradle clean test` with enabled logging just in order to demonstrate it. Its output contains FQN

**Comment by sksamuel** [comment]
@yevhenii8 There is a fix included in 4.2.0.RC2

However you can only use --tests to filter down to the class level. You can't include a test name. This is because gradle tries to turn them into method handles, and these aren't methods.

So, `./gradlew test --tests com.foo.MyTest` works but `./gradlew test --tests com.foo.MyTest.should do this` will not.
